bootstrap/src/hooks/lastProject.js

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from "react";
+
+import sdk from "../sdk";
+
+export const useLastProject = () => {
+    const [lastProjectId, setLastProjectId] = useState(null);
+    useEffect(() => {
+        const getLastProject = async () => {
+            try {
+                const projectId = await sdk.project.getCurrentProjectId();
```

Comment:
I think we agreed to not use this, since the user has no way of setting this from the app.

Reply:
Right, both the hooks (lastProject and projectAvailability) should not be there.